0_locs_poly_setup/src/select_polygons_by_points.R

```diff
@@ -0,0 +1,25 @@
+#' Function to collate multiple {sf} polygon files together, load points as a {sf} 
+#' object from a tibble, and select the polygons that intersect with the points.
+#' 
+#' @param shapefiles a list of {sf} polygon files
+#' @param points a tibble of locations with columns Latitude and Longitude in
+#' EPSG:4326 (WGS 84)
+#' @returns filepath for new {sf} file that contains the polygons that contained 
+#' points
+#' 
+#' 
+select_polygons_by_points <- function(shapefiles, points){
+  # load and collate all NHD polygons
+  shps <- map(shapefiles, read_sf) %>% 
```

Comment:
suggestion: I think `map_df` would do the same thing here without needing `bind_rows()`

Reply:
`map_dfr` does indeed do this... but apparently that is superseded with recent stupid tidverse updates... I like map_dfr and am personally VERY conflicted by this change. I'm going to change it to map_dfr() because it's more efficient and now I know that you, and MR and KW all also think this new update is dumb. ;)